src/test/scala/com/microsoft/ml/spark/explainers/LassoRegressionSuite.scala

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.explainers
+
+import breeze.linalg.{norm, DenseMatrix => BDM, DenseVector => BDV}
+import breeze.numerics.abs
+import com.microsoft.ml.spark.core.test.base.TestBase
+
+class LassoRegressionSuite extends TestBase {
+  // The following test can be verified in python with following code:
+  // from sklearn import linear_model
+  // X = np.array([[1, 1], [2, -2], [3, 3], [4, 5]])
+  // y = np.array([1.01, 1.98, 3.03, 4.05])
+  // clf = linear_model.Lasso(alpha=0.05, fit_intercept = True)
+  // clf.fit(X, y)
+  // print(clf.coef_)
+  // print(clf.intercept_)
+  test("LassoRegression should regress correctly without weights with intercept") {
+    val a = BDM((1.0, 1.0), (2.0, -2.0), (3.0, 3.0), (4.0, 5.0))
+    val b = BDV(1.01, 1.98, 3.03, 4.05)
+
+    val result = new LassoRegression(0.05d).fit(a, b, fitIntercept = true)
+
+    assert(norm(result.coefficients - BDV(0.94072731, 0.02135768)) < 1e-3, "coefficients disagreed")
```

Comment:
Perhaps we can use tolerant equality in scalatest for these

Reply:
done.